src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-update.component.ts

```diff
@@ -63,6 +64,14 @@ export class TextExerciseUpdateComponent implements OnInit {
         private router: Router,
     ) {}
 
+    get editType(): EditType {
```

Comment:
Small remark: this is duplicated in `modeling-exercise-update.component`. you can technically extract this function into `exercise-utils` and do something like:

```ts
export function getEditTypeOfExercise(isImport: boolean, exerciseId? : number): EditType {
   if (this.isImport) {
    return EditType.IMPORT;
  }

  return exerciseId ? EditType.CREATE : EditType.UPDATE; 
}
```

Reply:
Hi Ivan, you are right about the duplicated code.
However, not only this piece of code is duplicated in these text-/modeling-exercise-update-components.
That is why I think it's better that I do the complete refactoring of these components in a separate PR.